testlist/sim.dat

```diff
@@ -1,11 +1,4 @@
 /sim
--sim:cxxtest
-
-# CONFIG_SIM_M32=y
-# The recent versions of macOS is 64-bit only
--sim:loadable
--sim:module32
--sim:sotest32
 
 # X11
```

Comment:
I have left the X11 ones here because I don't think we have the requirements in the Docker container to support this yet.